docs/root/configuration/overview/v2_overview.rst

```diff
@@ -0,0 +1,774 @@
+.. _config_overview_v2:
```

Comment:
This is a merge issue. Please merge your new docs in this section into the new doc structure. Sorry for the churn.

Reply:
Yes, just noticed this. Thanks.